### PR TITLE
Fix #31 - Update about pages

### LINKED ISSUE #31: Update about pages
To remove no longer working links and links to the old repository.

### PR DESCRIPTION
**Pull request by Lineflyer**
- Some formatting
- Remove dead links and outdated information
- Direct current development to c:geo team repo
- Remove outdated cz files for now (needs preferably be added to crowdin later on for translation)
- Tried to remove the czech language exception for language variable. @cgeo/team Please kindly review my attempt to code :)

### CODE REVIEW

File: src/main/assets/en_about.html [hunk 1]
```diff
@@ -1,36 +1,32 @@
-Web page
-<br/>&nbsp;&nbsp;<a href="https://groups.google.com/d/forum/whereyougo">https://groups.google.com/d/forum/whereyougo</a>
-<br/>&nbsp;&nbsp;<a href="mailto:whereyougo@googlegroups.com/">whereyougo@googlegroups.com</a>
-<br/>&nbsp;&nbsp;<a href="https://github.com/biylda/WhereYouGo/blob/master/PRIVACY.md">Privacy policy</a>
+<b>Related web pages:</b>
+<br/>
+<br/>&nbsp;&nbsp;<a href="https://groups.google.com/d/forum/whereyougo">Google Group mailing list webpage</a>
+<br/>&nbsp;&nbsp;<a href="mailto:whereyougo@googlegroups.com/">Direct mail to whereyougo@googlegroups.com</a>
```
Comment (bekuno) [inline src/main/assets/en_about.html]: Is "c:geo" now owner or manager of this google group?

File: src/main/assets/en_about.html [hunk 2]
```diff
@@ -1,36 +1,32 @@
...
+<small><a href="https://github.com/cgeo/WhereYouGo">https://github.com/cgeo/WhereYouGo</a>
 </small>
-<br/>&nbsp;&nbsp;<b>CachingFoX</b>
-<br/>&nbsp;&nbsp;&nbsp;&nbsp;
-<small><a href="https://code.google.com/r/cachingfox-whereyougo/">https://code.google.com/r/cachingfox-whereyougo</a>
-</small>
-<br/>&nbsp;&nbsp;<b>Jochen Sprickerhof (jspricke)</b>
-<br/><br/>
-Translation
+<br/>
```
Comment (bekuno) [inline src/main/assets/en_about.html]: change to 
<br /><br />
(missing newline)

File: src/main/java/menion/android/whereyougo/preferences/PreferenceValues.java [hunk 3]
```diff
@@ -181,11 +181,7 @@ public static String getLanguageCode() {
             lang = java.util.Locale.getDefault().getLanguage();
         }
         Logger.w(TAG, "getLanguageCode() - " + lang);
```
Comment (bekuno) [inline src/main/java/menion/android/whereyougo/preferences/PreferenceValues.java]: remove the lines:
~~~
        String lang = Preferences.getStringPreference(R.string.pref_KEY_S_LANGUAGE);
        if ("".equals(lang) || "default".equals(lang)) {
            lang = java.util.Locale.getDefault().getLanguage();
        }
        Logger.w(TAG, "getLanguageCode() - " + lang);
~~~
add lines
~~~
        // language for info dialog html files
        // only english supported
        Logger.w(TAG, "getLanguageCode() - " + "en");
~~~

File: src/main/assets/en_about.html (hunk 1 above)
Comment (Lineflyer) [reply src/main/assets/en_about.html]: Nope, this is a group which existed before.
I only subscribed to that group and set up a job to pull these mails to our support system, in order to be able to answer them.

We can think about setting up an own contact address at a later point of time.

**Comment by Lineflyer** [comment]
@bekuno Thanks!

**Pull request merged**